base/src/main/resources/META-INF/resources/frontend/styles/commons-demo/shared-styles.css

```
@@ -51,5 +51,122 @@ code-highlighter code {
+	right: 0.5rem;
+}
+
+[orientation="vertical"] [slot="primary"] vaadin-button.source-code-viewer-show-button {
+	top: 0.5rem;
+}
+
+[orientation="vertical"] [slot="secondary"] vaadin-button.source-code-viewer-show-button {
+	top: 0;
+}
+
+/* Rotate the icons to match the layout disposition.*/
```

Stylelint (17.14.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)